examples/simpleMVC/src/com/domwires/example/simplemvc/commands/GenerateNickNameCommand.as

```diff
@@ -0,0 +1,32 @@
+/**
+ * Created by CrazyFlasher on 1.12.2016.
+ */
+package com.domwires.example.simplemvc.commands
+{
+	import com.domwires.core.mvc.command.AbstractCommand;
+	import com.domwires.example.simplemvc.model.IAppModel;
+	import com.domwires.example.simplemvc.utils.RandomStringUtil;
+
+	public class GenerateNickNameCommand extends AbstractCommand
+	{
+		[Autowired]
+		public var model:IAppModel;
+
+		override public function execute():void
+		{
+			super.execute();
+
+            var nickname:String;
+            var random_name:Number;
+            var random_surname:Number;
+            var name:Array = new Array("Hayman ","Junious ","Nathaniel ","Alvin ","Lashley ","Rolandus ", "Rolandus", "Albany");
+            var surname:Array = new Array("Beemer","Ingles","Geffer","Pariet","Knowlton","Riddel", "Todd");
+            random_name = Math.floor(Math.random() * name.length);
```

Comment:
do not use "_" in local variables. Example according language conventions: "randomName"